src/components/TicketsModal/styled.ts

```diff
@@ -0,0 +1,43 @@
+import { Typography, Select } from "@mui/material";
+import Box from "@mui/material/Box";
+import styled from "styled-components";
+
+export const ModalBox = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Better just to create fullscreen container for a modal, and just center its content(modal) with flex. Why?
Because in this case you'll be able to:
1) block page content and interactions with this container
2) `onClick` handler for modal container, that, for example, can handle `onClose` event.

Reply:
Thx, done.